src/System.Windows.Forms/tests/UnitTests/System.Windows.Forms.Tests.csproj

```
@@ -24,6 +27,10 @@
     <ProjectReference Include="..\TestUtilities\System.Windows.Forms.TestUtilities.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <NativeProjectReference Include="WindowsFormsTests\CMakeLists.txt" CMakeProject="WindowsFormsTests\WindowsFormsTests.proj" BuildNative="true" />
```

Can we rename to align with https://github.com/dotnet/winforms/tree/main/src/System.Windows.Forms/tests/InteropTests/NativeTests?
```suggestion
    <NativeProjectReference Include="WindowsFormsTests\CMakeLists.txt" CMakeProject="NativeTests\NativeTests.proj" BuildNative="true" />
```

---

It did not work for me. Have  issues with CMake SDK where both projects use `NativeTests` as output folder. Maybe `NativeTests\WindowsFormsTests.proj` is good compromise. Not sure that this will works.